Releases/v1.1.0/docker/.gitignore

```diff
@@ -0,0 +1 @@
+requirements.txt
```

Comment:
Is this file used in the docker file? If it is , it should be included in the repository.